src/Evolution/Executables/ForceFree/EvolveForceFree.hpp

```diff
@@ -38,6 +38,7 @@
 #include "Evolution/Systems/ForceFree/BoundaryCorrections/Factory.hpp"
 #include "Evolution/Systems/ForceFree/BoundaryCorrections/RegisterDerived.hpp"
 #include "Evolution/Systems/ForceFree/ElectricCurrentDensity.hpp"
+#include "Evolution/Systems/ForceFree/RecoverPhysicalVariables.hpp"
```

Comment:
How about the name from your PR title: `EmVariables.hpp` or `ElectromagneticVariables.hpp` or `MaxwellVariables.hpp`

Reply:
Hmm I'm not sure they are clear enough since technically densitized variables can still be considered as EM variables.

Reply:
Will it be too overkill to call these variables "primitives"? Also any idea on this @ermost ?